merge_hours.py

```diff
@@ -0,0 +1,117 @@
+import pandas as pd
+
+
+def flatten_list(to_flat):
```

Comment:
numpy.flatten() does the same:
https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.ndarray.flatten.html